client/src/constants/participantTableConstants.js

```diff
@@ -129,9 +129,10 @@ const columns = {
   callbackStatus: { id: 'callbackStatus', name: 'Callback Status', sortOrder: 14 },
   userUpdatedAt: { id: 'userUpdatedAt', name: 'Last Updated', sortOrder: 15 },
   engage: { id: 'engage', name: null, sortOrder: 16 },
-  edit: { id: 'edit', name: null, sortOrder: 17 },
+  edit: { id: 'edit', name: null, sortOrder: 21 },
```

Comment:
total nit, maybe shift this value to the end of the list

Reply:
This is in object format ... technically position is irrelevant but for consistency purpose we can change this.